ckan/templates/user/edit_user_form.html

```diff
@@ -24,7 +24,7 @@
 
   <fieldset>
     <legend>{{ _('Change password') }}</legend>
-    {{ form.input('old_password', type='password', label=_('Old Password'), id='field-password', value=data.oldpassword, error=errors.oldpassword, classes=['control-medium'], attrs={'autocomplete': 'off'} ) }}
+    {{ form.input('old_password', type='password', label=_('Sysadmin Password') if c.is_sysadmin else _('Old Password'), id='field-password', value=data.oldpassword, error=errors.oldpassword, classes=['control-medium'], attrs={'autocomplete': 'off'} ) }}
```

Comment:
Funny you say that. I actually had done that but then saw that it was not done in any of the other lines so reverted it. But I agree that it would be much more readable. Changing it back now.